test/index.js

```diff
@@ -38,10 +38,21 @@ test('decode: tag 0', (t) => {
   );
 });
 
-test('decode: tagCode 0b11111', (t) => {
-  t.throws(() => {
-    asn1Tree.decode(b(tag(CLS_UNIVERSAL, FORM_PRIMITIVE, 0b11111), 0));
-  }, 'Extended tags are not supported');
+test('decode: tagCode 0x9F32 (high tag)', (t) => {
```

Comment:
high tag -> extended tag?

Reply:
In ASN.1 it's called low-tag and high-tag. I believe a high-tag or extended tag is interchangeable.

Reply:
Good to know!